### DISCUSSION TITLE
How to wrap long lines in linear view?

### DISCUSSION THREAD

**Discussion by evilpan** [Vector35/binaryninja-api#6023] [Q&A]
In Linear View when reading HLIL or Pseudo C, there's some code with long statement:
![img](https://github.com/user-attachments/assets/1960b088-3939-443b-b26e-eaa996a5b58a)

Currently we can use the scrollbar to navigate the code but it's not very convenient. Is there any settings that can set max line width in the Linear view? Or even better use IDE-like linter to format the pseudo code.

Thanks.

**Comment by psifertex** [comment]
This is a feature request we're tracking via https://github.com/Vector35/binaryninja-api/issues/2366

Make sure to upvote and subscribe to that issue to be notified when it's resolved on dev.
